packages/solo/src/set-fake-chain.js

```diff
@@ -29,16 +29,18 @@ export default function setFakeChain(basedir, GCI, fakeDelay) {
     }
   };
 
-  JSON.parse(fs.readFileSync(fn)).forEach(add);
+  for (const conn of JSON.parse(fs.readFileSync(fn))) {
+    add(conn);
+  }
   const newconn = {
     type: 'fake-chain',
     GCI,
     fakeDelay,
   };
   add(newconn);
   const connections = [];
-  Object.entries(connsByType).forEach(([_type, conns]) =>
-    connections.push(...conns),
-  );
+  for (const [_type, conns] of Object.entries(connsByType)) {
+    connections.push(...conns);
+  }
```

Comment:
For consistency with the analogous change in packages/solo/src/set-gci-ingress.js:
```suggestion
  for (const conns of Object.values(connsByType)) {
    connections.push(...conns);
  }
```